### ISSUE TITLE
OverflowException in SucceededRenderer

### ISSUE THREAD

**Issue by SergeyA** [HangfireIO/Hangfire#367]
SucceededRenderer uses int.Parse() to parse Latency value, but Latency is Long. 
Therefore an OverflowException occurs for tasks created about 1 month ago.

Stack Trace: 
```
[OverflowException: Value was either too large or too small for an Int32.]
   System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info) +395
   Hangfire.Dashboard.JobHistoryRenderer.SucceededRenderer(HtmlHelper html, IDictionary`2 stateData) +155
   Hangfire.Dashboard.JobHistoryRenderer.RenderHistory(HtmlHelper helper, String state, IDictionary`2 properties) +54
   Hangfire.Dashboard.Pages.JobDetailsPage.Execute() +3481
   Hangfire.Dashboard.RazorPage.TransformText(String body) +41
   Hangfire.Dashboard.RazorPageDispatcher.Dispatch(RequestDispatcherContext context) +239
   Hangfire.Dashboard.<>c__DisplayClass8.<UseHangfireDashboard>b__4(IDictionary`2 env) +535
   Microsoft.Owin.Infrastructure.AppFuncTransition.Invoke(IOwinContext context) +41
   Microsoft.Owin.Mapping.<Invoke>d__0.MoveNext() +611
   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +144
   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +84
   Microsoft.Owin.Host.SystemWeb.IntegratedPipeline.<RunApp>d__5.MoveNext() +291
   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +144
   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +84
   Microsoft.Owin.Host.SystemWeb.IntegratedPipeline.<DoFinalWork>d__2.MoveNext() +293
   System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() +32
   Microsoft.Owin.Host.SystemWeb.IntegratedPipeline.IntegratedPipelineContext.EndFinalWork(IAsyncResult ar) +208
   System.Web.AsyncEventExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute() +443
   System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) +288
```

**Comment by odinserj** [comment]
Hi @SergeyA, would you like to create a PR agains the `master` branch?

**Comment by odinserj** [comment]
Closing, the issue was fixed in #372.
